LONDON| MAY_2025 |Sisay Mehari | Form Control

Sisu860 opened the pull request:
LONDON| MAY_2025 |Sisay Mehari | Form Control

## Form Controls Project Submission

This form is for ordering a T-shirt. It collects only necessary data from users, using semantic and accessible HTML.

---

### ✅ Self-Check Criteria
- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

cjyuan reviewed (commented):
Code is error free and well-indented. Form looks great too. Nice job!

Currently a user can enter a name consisting of only space characters (e.g., " "). Can you enforce a stricter validation rule via the `pattern` attribute to disallow any name that contains only space characters.

---
In your PR description, instead of including the checklist from the project's `README.md`, you can check off the 3rd and 5th items in the original Self-check list to confirm that your implementation meets all the requirements outlined in the project's `README.md`.

I restored the original checklist in your PR description. can you check them to make them look like
![image](https://github.com/user-attachments/assets/6f85e6c5-4dd7-49aa-a9fc-e17a9dc2189d)
?

Sisu860 commented:
Thanks for the feedback! I’ve added the pattern to the name input so it won’t accept just spaces anymore.
Also updated the PR description checklist like you showed me.
Let me know if there’s anything else I should fix!

cjyuan commented:
Changes look good!
